File: UNIDAD 3/JAVA/U3_P3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.u3_p3; //1
import java.util.Scanner; //1
/**
 *
 * @author LaboratorioLTI2
 */
public class U3_P3 { //1

    public static void main(String[] args) { //2
        Scanner sc = new Scanner(System.in); //1
        int dia; //2
        System.out.print("Escrible el dia de la semana numerico: "); //3
        dia = sc.nextInt(); //4
        switch (dia){ //5
        case 1:
        System.out.println("Lunes");
        break;
        case 2:
        System.out.println("Martes");
        break;
        case 3:
        System.out.println("Miercoles");
        break;
        case 4:
        System.out.println("Jueves");
        break;
        case 5:
        System.out.println("Viernes");
        break;
        case 6:
        System.out.println("Sabado");
        break;
        case 7:
        System.out.println("Domingo");
        break;
        default:
            System.out.println("No existe el dia ingresado");
        } //6
        
    } //6
    
} //6

File: UNIDAD 3/C#/U3_P3.cs
using System;

namespace U3_P3
{
    internal class U3_P3
    {
        static void Main(string[] args)
        {

            int dia; //2
            Console.Write("Escrible el mes del año numerico: "); //3
            dia = Convert.ToInt32(Console.ReadLine()); //4
            switch (dia)
            { //5
                case 1:
                    Console.WriteLine("Lunes");
                    break;
                case 2:
                    Console.WriteLine("Martes");
                    break;
                case 3:
                    Console.WriteLine("Miercoles");
                    break;
                case 4:
                    Console.WriteLine("Jueves");
                    break;
                case 5:
                    Console.WriteLine("Viernes");
                    break;
                case 6:
                    Console.WriteLine("Sabado");
                    break;
                case 7:
                    Console.WriteLine("Domingo");
                    break;

                default:
                    Console.WriteLine(" Erorr!! No existe el dia ingresado");
                    break;

            }
        }
    }
}